settings.gradle

```diff
@@ -10,6 +10,7 @@ include 'mongodb'
 include 'debezium'
 include 'twitter'
 include 'xa-test'
+include 'jet-spring-boot-starter'
```

Comment:
I think we should rename module to `spring-boot-starter-jet` to follow the naming convention. See https://github.com/spring-projects/spring-boot/tree/master/spring-boot-project/spring-boot-starters

Reply:
Even though community starters do not follow any convention as far as I see.

Reply:
there is a convention actually https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-starter

see the warning in the readme here https://github.com/spring-projects/spring-boot/tree/master/spring-boot-project/spring-boot-starters

```
https://github.com/spring-projects/spring-boot/tree/master/spring-boot-project/spring-boot-starters
```

Reply:
Oh, I've missed that. Alright, we are compliant then :)

Reply:
Maybe `hazelcast-jet-spring-boot-starter`? _hazelcast jet_ is what ppl will search for the starter.

Reply:
I mean in IDE when you fill out the dependency artifact id.